packages/atomic/cypress/fixtures/test-fixture.ts

```diff
@@ -0,0 +1,129 @@
+type SearchInterface = HTMLElement & {
+  initialize: (opts: {
+    accessToken: string;
+    organizationId: string;
+  }) => Promise<void>;
+  executeFirstSearch: () => void;
+};
+
+export type TestFeature = (e: TestFixture) => void | Promise<void>;
+
+export type TagProps = Record<string, string>;
+
+export class TestFixture {
+  private urlParams: {key: string; value: string}[] = [];
+  private aliases: TestFeature[] = [];
+  private testURL = 'http://localhost:3333/pages/test.html';
+  private execFirstSearch = true;
+
+  constructor() {
+    cy.visit(this.testURL).injectAxe();
+    this.intercept();
+
+    cy.document().then((doc) => {
+      const searchInterface = doc.createElement(
+        'atomic-search-interface'
+      ) as SearchInterface;
+      doc.body.appendChild(searchInterface);
+      cy.get('atomic-search-interface').as(this.elementAliases.SearchInterface);
+    });
+  }
+
+  public with(feat: TestFeature) {
+    feat(this);
+    return this;
+  }
+
+  public withURLParam(key: string, value: string) {
+    this.urlParams.push({key, value});
+    return this;
+  }
+
+  public withoutFirstAutomaticSearch() {
+    this.execFirstSearch = false;
+    return this;
+  }
+
+  public withAlias(aliasFn: TestFeature) {
+    this.aliases.push(aliasFn);
+    return this;
+  }
+
+  public init() {
+    document.location.hash = `#${this.urlParams.reduce(
+      (hash, param, i) =>
+        `${hash}${i === 0 ? '' : '&'}${param.key}=${encodeURIComponent(
+          param.value
+        )}`,
+      ''
+    )}`;
+
+    cy.get(`@${this.elementAliases.SearchInterface}`).then(($si) => {
```

Comment:
I'm not familiar with this convention with the dollar sign, what is it?

Reply:
It's for jQuery element (jQuery global variable being `$`)